analysis/study_definition_ethnicity.py

```diff
@@ -0,0 +1,44 @@
+from cohortextractor import (
+  StudyDefinition,
+  patients,
+  codelist,
+  codelist_from_csv,
+  Measure,
+)
+
+from datetime import date
+
+end_date = "2022-03-01"
+
+from codelists import *
+  
+study = StudyDefinition(
+    default_expectations={
+      "date": {"earliest": "1900-01-01", "latest": "today"},
+      "rate": "uniform",
+    },
+    
+    index_date=end_date,
+    
+    population = patients.all(),
+    
+    # Ethnicity
+    eth2001 = patients.with_these_clinical_events(
```

Comment:
Any reason for not including the 16 categories?

(sorry for fly by)

Reply:
I was thinking there may be small numbers in some of those categories but I'm guessing by your question that you think it's worth extracting all 16 :) (and I suppose it can always be aggregated later if necessary).

Reply:
Yes I do. Two reasons
- I think the numbers will be large enough (for some groups we may combine/suppress/aggerate over longer period)
- We have seen very distinct inequalities in other work between groups that are obfuscated when we do 6 e.g. Indian v Pakistani origin v "South Asian"